File: b_rank_skillcheck_archive/beam_reflecting_01.py
# 実行時間 10ms
h, w = list(map(int, input().split()))

for _ in [0] * h:
    s = input()
    l = [0] * w

    for i in range(w):
        if s[i] == "_":
            l[i] = 0
        elif s[i] == "/":
            l[i] = 1
        elif s[i] == "\\":
            l[i] = 2
    print(*l)

File: b_rank_skillcheck_archive/beam_reflecting_01.cs
// 実行時間 20ms
using static System.Console;
using static System.Linq.Enumerable;

internal class Program
{
    static void Main()
    {
        int[] hw = ReadLine().Split().Select( int.Parse ).ToArray();

        SetOut( new System.IO.StreamWriter( OpenStandardOutput() ) { AutoFlush = false } );
        foreach( int _ in Range( 0, hw[0] ) ) {
            byte[] b = new byte[hw[1]];
            string s = ReadLine();

            foreach( int i in Range( 0, hw[1] ) ) {
                switch( s[i] ) {
                case '_':
                    b[i] = 0;
                    break;
                case '/':
                    b[i] = 1;
                    break;
                case '\\':
                    b[i] = 2;
                    break;
                }
            }
            WriteLine( string.Join( " ", b ) );
        }
        Out.Flush();
        return;
    }
}
